semgrep_output_v1.atd

```diff
@@ -820,6 +854,47 @@ type ci_scan_results <ocaml attr="deriving show"> = {
 
 
 
+type parsing_stats <ocaml attr="deriving show"> = {
```

Comment:
I don't know a lot about atd, but I think it supports includes right? Would it make sense to split the product stuff out into domain specific files and include things?

Reply:
The current ATD does not support include (it was originally a design decision of @mjambon. Since then martin
started to add support for includes/modules, but it's not merged yet.
But yes would be good to split semgrep_output_v1.atd in multiple files at some point.

Reply:
It supports include between records within the same file. As far as I can tell support for imports across files is only available when generating ocaml, it doesn't work in python: https://atd.readthedocs.io/en/latest/atdgen.html#modularity-referring-to-type-definitions-from-another-atd-file

Reply:
I was referring to: https://github.com/ahrefs/atd/pull/297